paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -98,66 +116,109 @@ def get_reviewers(filename):
     return authors[:3]
 
 
-def review(filename):
+def review(filename, description, over):
     reviewers = ' '.join(get_reviewers(filename))
-    description = (
-        'This change was made automatically by the perf_update_soaconfigs script. If not reviewed'
-        ' in a week, it will be merged by the perf or paasta teams. For more context on why these'
-        ' changes are being made see: PERF-2439'
-    )
     subprocess.check_call((
-        'review-branch',
+        'git',
```

Comment:
This is way too dangerous. 

Does it need to push at all?

Reply:
(no)